### ISSUE TITLE
In the chapter What is Addition, when talkback is turned on along with the comments a sound is heard in question page

### ISSUE THREAD

**Issue by KolliAnitha** [oppia/oppia-android#4370]
**Describe the bug**
In the chapter What is Addition, when talkback is turned on along with the comments a sound is heard in question page

**To Reproduce**
1. Go to 'Addition and Subtraction' - What is Addition
2. Click on Continue to go to the next pages
3. In the question page Which of the following pictures shows a total of seven people 'Choose all the possible options' Focus on any of the options.
4. Along with the option number a sound is heard

**Expected behavior**
No extra sound should be heard when the focus is over the options

https://user-images.githubusercontent.com/101634267/169806027-de038d14-4332-4298-ad36-214744779ca7.mp4

Environment:
Device name: One plus Nord2 5G
Android version : Android 11
App version: 0.7-alpha-6c08d882d3

**Comment by rt4914** [comment]
@KolliAnitha This looks like correct behaviour. I will try to explain how it works and maybe based on that you can correct me.

<img src="https://github.com/oppia/oppia-android/assets/9396084/f54669e2-0a4a-47ab-b800-a8f7f0059194" width="250" />

If you have a look at the screenshot above the text says `not Checked, Check box, Blue, 4 of 6`. So basically there are four parts:
- `not Checked` - mentions the state of checkbox - **NOT CONTROLLED BY OUR CODE**
- `Check box` - mentions the type of element - **NOT CONTROLLED BY OUR CODE**
- `Blue` - text associated with Checkbox - This we can control but currently its automatic.
-  `4 of 6` - indexing by recyclerview and talkback - **NOT CONTROLLED BY OUR CODE**

So basically we are actually not doing anything here and everything is controlled by screen-reader only. And this works in similar fashion on other apps too. 
That's why I think it is the expected behaviour because user will find this same experience across all other apps too.

Let me know your thoughts.

**Comment by KolliAnitha** [comment]
@rt4914, there was a beep sound coming in between the options. It is working fine now. I am closing this issue.

**Comment by adhiamboperes** [comment]
We closed this but for future reference the beep sound is caused by device settings--if you enable touch feedback on your device you should still be able to hear it.
